themes/default/content/case-studies/lykke.md

```diff
@@ -0,0 +1,67 @@
+
+## Solution: Pulumi Crosswalk and Infrastructure-as-Code
+
+Lykke engineering head Damian Hickey had been aware of Pulumi since the company launched in 2017 and decided to deploy the new infrastructure using [Pulumi Crosswalk for AWS](/crosswalk/aws/). This allowed him to design and manage infrastructure within his development team, rather than building a dedicated ops team. Crosswalk also provided Lykke with built-in Well Architected patterns and best practices - helping the team move new applications into production faster.
+
+With Pulumi, Damian’s team quickly set up their infrastructure without having to learn a new, proprietary programming language. In addition to creating their own abstractions, models, and best-practice files, Lykke was also able to take advantage of pre-built templates and libraries available right out of the box.
+
+Lykke is now running multiple accounts with many core services each - all defined and managed with Pulumi. Pulumi Crosswalk allows the team to dynamically spin up instances through a single pull request within the stack and manage across multiple environments as needed.
+
+## Results
+
+One of the biggest considerations for the team was cost management. Pulumi helped Lykke adopt cloud-native architectures faster and migrate to managed services like Amazon Elastic Kubernetes Service (EKS) . This helped the Lykke team to run their services more efficiently, reducing costs by at least 20% and making their services more scalable.
+
+Pulumi Crosswalk also helped the team to configure services like Amazon VPC using only a dozen lines of code - something that required 10x more configuration with other IaC tools. The code review process also became faster, with infrastructure changes taking hours, rather than days to deliver.
+
+Within a month of adopting Pulumi, Lykke deployed core supporting infrastructure for a new product. By improving overall productivity for their development team, Lykke was able to deliver the new product to production customers in a matter of months. Lykke is also adopting a comprehensive approach to infrastructure policy using Pulumi’s CrossGuard framework. This ensures that internal security, compliance and cost management policies are automatically enforced through policy-as-code.
```

Comment:
```suggestion
Within a month of adopting Pulumi, Lykke deployed core supporting infrastructure for a new product. By improving overall productivity for their development team, Lykke was able to deliver the new product to production customers in a matter of months. Lykke is also adopting a comprehensive approach to infrastructure policy using Pulumi’s [CrossGuard](/crossguard/) framework. This ensures that internal security, compliance and cost management policies are automatically enforced through policy-as-code.
```

Reply:
We should use a relref here (and with any others as well).